### PR TITLE
Events: Add outbound detail protobufs, request status code, and update event encrypt filter.

### PR DESCRIPTION
**Pull request by jimlambrt**
- feat (events): Add status code to audit events
- feat (events): Add the resp proto msg to every audit event
- chore (events): Update eventlogger dep
- feat (events): Begin to add tagging to response proto msgs

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -137,6 +137,12 @@ protobuild:
 	@protoc-go-inject-tag -input=./internal/credential/store/credential.pb.go
 	@protoc-go-inject-tag -input=./internal/credential/vault/store/vault.pb.go
 
+
```
Comment (louisruch) [inline Makefile:140]: Is there a reason you want all this white space around these? If they are being isolated perhaps a comment as on line 146...

**Review by louisruch — COMMENTED** [review]
So great to see this in action!

File: internal/servers/controller/handlers/outgoing_interceptor.go [hunk 2]
```diff
@@ -20,13 +21,21 @@ const (
...
 	m = m.ProtoReflect().Interface()
 	if !m.ProtoReflect().IsValid() {
 		// This would be the case if it's a nil pointer
 		w.WriteHeader(http.StatusNoContent)
 		return nil
 	}
+
+	// add the outgoing resp proto.  See: common.flushGatedEvents is where the
+	// event.Response.StatusCode is added.
+	if err := event.WriteAudit(ctx, "handlers.OutgoingInterceptor", event.WithResponse(&event.Response{Details: m})); err != nil {
```
[the remark this answers is not in the document]
Comment (jimlambrt) [reply internal/servers/controller/handlers/outgoing_interceptor.go:35]: ty

File: Makefile (hunk 1 above)
Comment (jimlambrt) [reply Makefile:140]: fair enough.  adding comment.

**Review by louisruch — DISMISSED** [review]
lgtm

**Pull request merged**